docs/local-testing.md

```diff
@@ -86,10 +101,14 @@ This deploys the challenge and healthcheck to the local Kubernetes cluster.
 
 ### Connect to the challenge
 
+Use the following command to connect to the challenge:
 ```
-kubectl port-forward deployment/test-1 :1337 &
+kubectl port-forward --namespace=[name_of_the_challenge] deployment/chal :1337 &
 ```
 
+Example: If you ran the challenge sample in the `samples/apache-php` folder, specify `apache-php` as the namespace. After forwarding completes, you can access `127.0.0.1:external_port` from your browser and should see a page similar to this one: 
```

Comment:
I think this part would be better like this:
```suggestion
Example: If you ran the challenge sample in the `samples/apache-php` folder, specify `apache-php` as the namespace. After you run the port-forward command, it will be shown in the command line from which external port the challenge is being forwarded. With that information, you can access `127.0.0.1:external_port` from your browser and should see a page similar to this one: 
```